packages/select/src/ux-select.ts

```diff
@@ -261,15 +264,19 @@ export class UxSelect implements UxInputComponent {
   }
 
   private unsetupListAnchor() {
-    this.listAnchor = null;
     this.winEvents.disposeAll();
   }
 
-  public listAnchor: { x: number | string, y: number | string } | null;
-  private calcAnchorPosition() {
-    const elDim = this.element.getBoundingClientRect();
-    const offsetY = (48 - elDim.height) / 2;
-    this.listAnchor = { x: elDim.left, y: elDim.top - offsetY };
+  private setListAnchorPosition() {
+    this.positioningFactory(
```

Comment:
calling the factory function will instantiate a new instance of `Positioning` every time. Is this intentional?

Reply:
I borrowed the approach from model demo
https://github.com/aurelia/ux/blob/b19034779fa081ca5ce744eafa4cb44c0dc723b2/app/src/modal-page.ts#L110

Moved to `attached`